js/src/wcdl/subsection/index.scss

```diff
@@ -1,5 +1,5 @@
 .wcdl-subsection {
-	&:not(:last-child) {
-		margin-bottom: var(--main-gap);
+	&:not(:first-child) {
+		margin-top: var(--main-gap);
```

Comment:
:grey_question: 
Why do we need to change it?
:nail_care: 
Since we already dropped IE support, WDYT of using `gap` in `.wcdl-section` and removing this file completely?

Reply:
> Why do we need to change it?

![2021-09-23 10 49 14](https://user-images.githubusercontent.com/17420811/134448058-0f860b2b-0723-49c9-b906-befab2af8ae8.png)

I want to add spacing to the circled position which is written [here](https://github.com/woocommerce/google-listings-and-ads/blob/d9bd2c0679d45c4152db21c049351e5320c78134/js/src/components/contact-information/phone-number-card/edit-phone-number-content.js#L120-L121). After searching the codebase, we don't have any instance that another element is placed after a `<Subsection>` currently. So I changed the specified spacing from bottom to top reversely.



> Since we already dropped IE support, WDYT of [using `gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap#browser_compatibility) in `.wcdl-section` and removing this file completely?

That's a good idea. However, the components under `wcdl` are currently composed by consumer sides, if we use flex/grid + gap, we have to do a relatively larger rewrite. On the other hand, all `<Subsection>` are currently placed inside `<Section.Card.Body>`, so if we convert `<Section.Card.Body>` to flex/grid + gap blocks directly, there would be many layout changes. I think we could open an issue to record this.

Reply:
Oh, I missed that we use `@wordpress/components.CardBody` in the middle :(
Changing WP components is not gonna happen anyhow soon or easily. Plus, for WP components, we cannot assume no-IE, so I think it's not even worth tracking here, as it's too tangled in legacy, and distant codebase.